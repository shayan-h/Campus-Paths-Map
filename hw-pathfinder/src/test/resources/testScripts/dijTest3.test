# Find a path with Dijkstra's algorithm when node is not in graph.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 10.0

FindPath simpleGraph A C