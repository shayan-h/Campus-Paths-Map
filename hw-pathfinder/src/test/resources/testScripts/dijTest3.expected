# Find a path with Dijkstra's algorithm when node is not in graph.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 10.000 from A to B in simpleGraph

unknown: C